python/pyspark/ml/functions.py

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
Is this file documented in PySpark API? I think we should add this module into `spark/python/docs/pyspark.ml.rst`